.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,135 @@
+name: Benchmark
+
+on:
+  pull_request_review:
```

Comment:
should be `pull_request_review_comment`?
```suggestion
  pull_request_review_comment:
```

Reply:
During the review discussion, many comments will be made.
For example, several `pull_request_review_comment ` events will be triggered during https://github.com/nervosnetwork/ckb/pull/2680#discussion_r632384482. It would be too noisy.

We can comment on `pull_request_review`. `pull_request_review` is enough.

Reply:
we have checked comment body here: `contains(github.event.review.body, '#benchmark')` I think it won't be triggered by other comments, right?

Reply:
by using `pull_request_review`, I think it will create a skipped action always:
<img width="1108" alt="WX20210514-181650@2x" src="https://user-images.githubusercontent.com/8990/118249632-932ff600-b4e0-11eb-9b76-78c5f58f7a52.png">

Reply:
It will be triggered but skipped.

> by using pull_request_review, I think it will create a skipped action always

Yeah, both `pull_request_review` and `pull_request_review_comment` trigger it.
Despite my desire to reduce the trigger events, pull_request_review versus pull_request_review_comment, the former wins.

If we don't want "Benchmark" to appear at PR status, we should choose `issue_comment` instead of `pull_request_*`

Reply:
pull_request_review: skip once

pull_request_review_comment: skip many times